stdlib/lexing.mli

```diff
@@ -108,6 +108,15 @@ val from_function : ?with_positions:bool -> (bytes -> int -> int) -> lexbuf
    starting at index 0, and return the number of bytes
    provided. A return value of 0 means end of input. *)
 
+val set_position : lexbuf -> position -> unit
+(** Set the initial tracked input position for [lexbuf] to custom value.
+   Does not set file (use [set_file] for it). Can be used to adjust location
+   information without need to traverse AST. *)
```

Comment:
```
@since 4.10
```

Reply:
More comments on the doc comment:

- "to **a** custom value" or just "Set the initial tracked input position for a [lexbuf]."
- If the API stands, it should be `{!set_filename}`, not `[set_file]`, although I'd change the whole sentence to something like "Ignores [pos_fname], see {!set_filename} for changing this field."
- The last sentence is not relevant to `Lexing` and I don't think should be here - `Lexing` has nothing to do with ASTs.